src/main/java/org/zalando/nakadi/service/StorageService.java

```diff
@@ -28,16 +35,40 @@
 @Service
 public class StorageService {
 
+    public static final String ZK_TIMELINES_DEFAULT_STORAGE = "/nakadi/timelines/default_storage";
     private static final Logger LOG = LoggerFactory.getLogger(StorageService.class);
-
     private final ObjectMapper objectMapper;
     private final StorageDbRepository storageDbRepository;
+    private final DefaultStorage defaultStorage;
+    private final CuratorFramework curator;
 
     @Autowired
     public StorageService(final ObjectMapper objectMapper,
-                          final StorageDbRepository storageDbRepository) {
+                          final StorageDbRepository storageDbRepository,
+                          @Qualifier("default_storage") final DefaultStorage defaultStorage,
+                          final ZooKeeperHolder zooKeeperHolder) {
         this.objectMapper = objectMapper;
         this.storageDbRepository = storageDbRepository;
+        this.defaultStorage = defaultStorage;
+        this.curator = zooKeeperHolder.get();
+    }
+
+    @PostConstruct
+    private void watchDefaultStorage() {
```

Comment:
It may happen that functionality that creates `/nakadi/timelines/default_storage` node will be called later than this `@PostConstruct` and in this case the watcher will fail to create so all updates of this node will be ignored.